pkg/api/vm/handler.go

```diff
@@ -255,6 +255,14 @@ func (h *vmActionHandler) migrate(ctx context.Context, namespace, vmName string,
 	if !vmi.IsRunning() {
 		return errors.New("The VM is not in running state")
 	}
+	for _, condition := range vmi.Status.Conditions {
+		if condition.Type == kv1.VirtualMachineInstanceReady {
+			if condition.Status != corev1.ConditionTrue {
+				return errors.New("The VM's Ready status is not True")
+			}
+			break
+		}
+	}
```

Comment:
Can we wrap this with a method, and /s/The VM's Ready status is not True/Can't migrate the VM, the VM is not in ready status.

Reply:
Updated it. Thank you.